docs/documentation/admin/workflows/admin_workflow_required_survey.md

```diff
@@ -0,0 +1,112 @@
+---
+id: admin_workflow_required_survey
+title: III. Procedural Surveys
+sidebar_label: III. Procedural Surveys
+---
+import useBaseUrl from '@docusaurus/useBaseUrl'; 
+import Highlight from '@theme/Highlight';
+
+<img alt="design" className="img_sizing" src={useBaseUrl('img/design/Workflows.svg')} />
+<br/>
+
+<div className="alert alert--secondary">
+
+## Summary {#summary}
+
+By default, _users_ can create a _task_ or change its state through the _tasks view_ section. However, _administrators_ may require _users_ to complete a _survey_ before **creating or changing the state of a task**. This feature – known as **procedural survey** – allows _administrators_ to:
```

Comment:
I think w need to distinguish between the creating and changing state.
The two main reasons
1.  Only creating (start-surveys) allows "external-surveys" 
2.  The configuration: start-surveys require the step "Create Task" , and change-state-surveys require the "Change Stage"  step